### PR TITLE
Mainnet Deploy 2019-12-19

### PR DESCRIPTION
**Pull request by ejwessel**
## Description

Quick description:

- [X] N/A
- [ ] New features
- [ ] Bug fixes
- [ ] Typo/small tweaks

## Changes

Implementation details. These are mainly to help your fellow developers.

### CODE REVIEW

File: source/delegate/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/delegate",
-  "version": "0.5.6-beta.2",
+  "version": "2.5.6",
```
Comment (poiresel) [inline source/delegate/package.json]: why did this bump from 0.5.6 -> 2.5.6?

Comment (ejwessel) [reply source/delegate/package.json]: fat fingering.

File: utils/pre-swap-checker/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pre-swap-checker",
-  "version": "0.0.1-beta.2",
+  "version": "1.0.1",
```
Comment (poiresel) [inline utils/pre-swap-checker/package.json]: this was not deployed so no need to increment.

File: source/delegate/package.json (hunk 1 above)
Comment (ejwessel) [reply source/delegate/package.json]: 1.5.6 is correct. will update.

File: utils/deployer/package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
   },
   "devDependencies": {
     "@airswap/tokens": "0.1.3",
-    "@airswap/order-utils": "0.3.15-beta.2",
+    "@airswap/order-utils": "1.3.15",
```
Comment (poiresel) [inline utils/deployer/package.json]: no contract was deployed so just increment patch version. 0.3.16.

**Pull request merged**